Makefile

```diff
@@ -32,3 +32,14 @@ test:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
 test-verbose:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race -v ./...
+
+docker-images: docks/build/Dockerfile docks/base/Dockerfile
+	docker build --rm -f ./docks/base/Dockerfile  --tag=goclang/base .
+	docker build --rm -f ./docks/build/Dockerfile --tag=goclang/build .
```

Comment:
goclang/build can be used to work with Clang 3.4 right? I am wondering if we should tag them with the Clang version?